src/aalimagecapturecontrol.cpp

```diff
@@ -54,6 +57,10 @@ AalImageCaptureControl::AalImageCaptureControl(AalCameraService *service, QObjec
     m_audioPlayer->setMedia(QUrl::fromLocalFile("/system/media/audio/ui/camera_click.ogg"));
     m_audioPlayer->setAudioRole(QAudio::NotificationRole);
 
+    char name[PROP_VALUE_MAX] = "";
+    property_get("ro.product.device", name, "");
```

Comment:
Will this be broken again under device-info?

Reply:
Yes